doc/sphinx/using/libraries/writing.rst

```diff
@@ -22,13 +22,17 @@ deprecated compatibility alias using :cmd:`Notation (abbreviation)`
    by a comma.
 
    This attribute is supported by the following commands: :cmd:`Ltac`,
-   :cmd:`Tactic Notation`, :cmd:`Notation`, :cmd:`Infix`.
+   :cmd:`Tactic Notation`, :cmd:`Notation`, :cmd:`Infix`, :cmd:`Ltac2`,
+   :cmd:`Ltac2 Notation`, :cmd:`Ltac2 external`.
 
    It can trigger the following warnings:
 
    .. warn:: Tactic @qualid is deprecated since @string__since. @string__note.
              Tactic Notation @qualid is deprecated since @string__since. @string__note.
              Notation @string is deprecated since @string__since. @string__note.
+             Ltac2 definition @qualid is deprecated since @string__since. @string__note.
+             Ltac2 alias @qualid is deprecated since @string__since. @string__note.
+             Ltac2 notation {+ @ltac2_scope } is deprecated since @string__since. @string__note.
```

Comment:
It just prints notation scopes?

Reply:
It prints the scope tokens, including terminals, so you'll get the shape of the notation.

Reply:
Oh, OK. I was just getting confused by the `ltac2_scope` name.